templates/Search/search.html.twig

```diff
@@ -11,6 +11,28 @@
     <div id="search-results-text">
       {{ "search.resultcount"|trans({'%count%': '<span></span>'}, "catroweb")|raw }}
     </div>
+    <div class = "circular-progress">
```

Comment:
might put it into its own file. We will need those spinners on many more sports on the website.
Just include it via twig, and if necessary provide means to initialize the template with IDs.

Take a look at snackbar.html.twig for example, which is included at  header_options.html.twig.